src/screens/DomainScreen/elements/Header/elements/ActionButtonGroup.js

```diff
@@ -1,148 +1,137 @@
-  globalButtonBlockContainer: {
-    paddingHorizontal: 0,
-    flex: 1
-  },
-  globalButtonFollowContainer: {
-    paddingHorizontal: 0,
-    flex: 1,
-    marginRight: 9
-  },
-  textButtonFollow: {
-    fontFamily: fonts.inter[600],
-    fontWeight: 'bold',
-    fontSize: normalizeFontSize(12),
-    lineHeight: normalizeFontSize(24),
-    color: COLORS.white
-  },
-  textButtonFollowing: {
-    fontFamily: fonts.inter[600],
-    fontWeight: 'bold',
-    fontSize: normalizeFontSize(12),
-    lineHeight: normalizeFontSize(24),
-    color: COLORS.holyTosca
-  },
-  unblockButtonText: {
-    fontSize: normalizeFontSize(12),
-    color: COLORS.white,
-    paddingHorizontal: 0
-  }
-});
+    blockButtonText: {
+        fontSize: normalizeFontSize(12),
+        color: '#FF2E63',
+        paddingHorizontal: 0,
+    },
+    buttonBlock: {
+        flex: 1,
+        height: normalize(36),
+        // width: normalize(88),
+        borderWidth: 0.5,
+        borderRadius: 8,
+        borderColor: '#FF2E63',
+        overflow: 'hidden',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: 'transparent',
+    },
+    buttonUnblock: {
+        flex: 1,
+        height: normalize(36),
+        // width: normalize(88),
+        borderWidth: 0.5,
+        borderRadius: 8,
+        borderColor: '#FF2E63',
+        overflow: 'hidden',
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: '#FF2E63',
+    },
+    buttonFollow: {
+        flex: 1,
+        // width: normalize(88),
+        height: normalize(36),
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        borderRadius: 8,
+        backgroundColor: colors.bondi_blue,
+        color: colors.white,
+    },
+    buttonFollowing: {
+        flex: 1,
+        // width: normalize(88),
+        height: normalize(36),
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        borderWidth: 1,
+        borderColor: colors.bondi_blue,
+        borderRadius: 8,
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'row',
+        flex: 1,
+    },
+    globalButtonBlockContainer: {
+        paddingHorizontal: 0,
+        flex: 1,
+    },
+    globalButtonFollowContainer: {
+        paddingHorizontal: 0,
+        flex: 1,
+        marginRight: 9,
+    },
+    textButtonFollow: {
+        fontFamily: fonts.inter[600],
+        fontWeight: 'bold',
+        fontSize: normalizeFontSize(12),
+        lineHeight: normalizeFontSize(24),
+        color: colors.white,
+    },
+    textButtonFollowing: {
+        fontFamily: fonts.inter[600],
+        fontWeight: 'bold',
+        fontSize: normalizeFontSize(12),
+        lineHeight: normalizeFontSize(24),
+        color: colors.bondi_blue,
+    },
+    unblockButtonText: {
+        fontSize: normalizeFontSize(12),
+        color: 'white',
+        paddingHorizontal: 0,
+    },
+})
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new `colors` import. This is a good practice as it maintains consistency across the codebase. However, it's important to ensure that the color values in the `colors` object match those previously used in `COLORS`.



<!-- This is an auto-generated comment by OSS CodeRabbit -->